### PR TITLE
Config dependabot for npm and pip package-ecosystems

### PR DESCRIPTION
**Pull request by mmorel-35**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,7 +1,91 @@
...
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: npm
+  directory: gh-actions/dispatch
+  schedule:
+    interval: daily
+  groups:
+    eslint:
```
Comment (phlax) [inline .github/dependabot.yml:23]: does this have the effect of batching dependabot prs??

if so - just wow - the coolest thing ive learnt this week!

Comment (phlax) [reply .github/dependabot.yml:23]: ah i see - new feature!

image

ill enable it now

**Review by phlax — APPROVED** [review]
lgtm, thanks @mmorel-35

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,7 +1,91 @@
...
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: pip
+  directory: bazel
```
Comment (phlax) [inline .github/dependabot.yml:89]: this one possibly should be bazel/website/requirements.txt

bazel/requirements.txt is empty and should probs be removed

Comment (phlax) [reply .github/dependabot.yml:89]: seems to be doing the right thing anyway

File: .github/dependabot.yml (hunk 1 above)
Comment (mmorel-35) [reply .github/dependabot.yml:23]: Yes ! You could even group every dependencies for each npm project using * pattern

**Pull request merged**